lib/rules/define-macros-order.js

```diff
@@ -130,6 +130,18 @@ function create(context) {
       },
       onDefineExposeExit(node) {
         defineExposeNode = getDefineMacrosStatement(node)
+      },
+
+      'CallExpression:exit'(node) {
+        // check if the node is a macro in the order
+        if (
+          node.callee &&
+          node.callee.type === 'Identifier' &&
+          order.includes(node.callee.name) &&
+          !ORDER_SCHEMA.has(node.callee.name)
```

Comment:
> Maybe the check could be more performant if there was no check against `ORDER_SCHEMA` in the condition? (removing the callbacks and the call to `has()`).

`Set.has()` is constant-time, so this is irrelevant to performance. But of course, there are multiple callbacks for the same element now, which adds some overhead. But since some of the handlers have some special logic (both in this file and [where they are defined](https://github.com/vuejs/eslint-plugin-vue/blob/f5cb93db8fd9ff40288406b6119e3c171b1ce66b/lib/utils/index.js#L1400-L1442)), we can't really change that.

Reply:
Okay, I got the point. I may be interested to do that after.